README.md

```diff
@@ -1,15 +1,19 @@
-# Kubeflow Blueprint
+# Kubeflow Blueprint On GCP (Google Cloud Platform)
```

Comment:
Note, the official branding for GCP is now `Google Cloud`, we should avoid `GCP` or `Google Cloud Platform` in user facing documentation (it's fine here, but would be nicer if we stick to the convention from the start).